sdks/python/apache_beam/testing/util.py

```diff
@@ -103,7 +103,7 @@ def expand(self, pcoll):
           | "ToVoidKey" >> Map(lambda v: (None, v)))
       _ = ((keyed_singleton, keyed_actual)
            | "Group" >> CoGroupByKey()
-           | "Unkey" >> Map(lambda (k, (_, actual_values)): actual_values)
+           | "Unkey" >> Map(lambda k___actual_values: k___actual_values[1][1])
```

Comment:
why double _ here?

Reply:
So one of the _s is from the original name (which indicated an ignored param). We could give it an explicit name or keep it with the _ to indicate unused.